docs/networking/linux-static-ip-configuration.md

```diff
@@ -200,19 +200,19 @@ You must create the `/etc/sysconfig/network-scripts/ifcfg-eth0` file.
 
   # Adding a public IP address.
   # The netmask is taken from the PREFIX (where 24 is Public IP, 17 is Private IP)
-  IPADDR0=12.34.56.78
-  PREFIX0=24
+  IPADDR=12.34.56.78
+  PREFIX=24
 
   # Specifying the gateway
-  GATEWAY0=12.34.56.1
+  GATEWAY=12.34.56.1
 
   # Adding a second public IP address.
-  IPADDR1=34.56.78.90
-  PREFIX1=24
+  IPADDR=34.56.78.90
+  PREFIX=24
 
   # Adding a private IP address.
-  IPADDR2=192.168.133.234
-  PREFIX2=17
```

Comment:
This doesn't create aliases.

If non-unique names are used, only the last-defined `IPADDR`, `PREFIX` will have any effect.